.github/workflows/validate_generic_master.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: R Package Validation report
+
+on: # Run this action when a release is published
+  push:
+    release:
+      types: [published]
+
+jobs:
+  r-pkg-validation:
+    name: Create report 📃
+    runs-on: ubuntu-latest
+    # Set Github token permissions
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    permissions:
+      contents: write
+      packages: write
+      deployments: write
+    steps:
+      - name: Checkout repo 🛎
+        uses: actions/checkout@v2
+
+      - name: Build report 🏗
+        uses: insightsengineering/thevalidatoR@main
```

Comment:
I think its normally best practice to tag releases then specify the releases as opposed to the general "main" branch which can update over time